src/js/burials/labels.jsx

```diff
@@ -14,8 +14,8 @@ export const locationOfDeathLabels = {
 };
 
 export const allowanceLabels = {
-  nonService: 'Non-service-connected death',
-  service: 'Service-connected death',
+  nonService: 'Non-service-connected death (for Veteran receiving a pension or disability compensation)',
+  service: 'Service-connected death (for Veteran death related to, or a result of, a service-connected disability)',
```

Comment:
Mmm...this is one of the finer points of grammar I'm not sure about. It _sounds_ like it should be 'as a result of', but I just copied @peggygannon's text from [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3836#issuecomment-314473982).

@peggygannon Should I change it the parenthetical statement to "(for Veteran death related to, or as a result of, a service-connected disability)", or leave it the way it is?